org.eclipse.scanning.test/src/org/eclipse/scanning/test/event/queues/processes/MonitorAtomProcessTest.java

```diff
@@ -115,7 +114,7 @@ public void testTermination() throws Exception {
 		pti.waitForBeanFinalStatus(5000l);
 		pti.checkLastBroadcastBeanStatuses(Status.TERMINATED, false);
 
-		Thread.sleep(100);
+		Thread.sleep(1000);
 //		TODO This is probably not a good way to test this as we can't guarantee what stage the terminate happens at
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)